cibuildwheel/util.py

```diff
@@ -150,11 +150,14 @@ def format_safe(template: str, **kwargs: Any) -> str:
             re.VERBOSE,
         )
 
-        # we use a lambda for repl to prevent re.sub interpreting backslashes
-        # in repl as escape sequences
+        # we use a function for repl to prevent re.sub interpreting backslashes
+        # in repl as escape sequences. Capturing the current value of value.
+        def repl_func(_: Any, value: Any = value) -> str:
+            return str(value)
+
         result = re.sub(
             pattern=find_pattern,
-            repl=lambda _: str(value),
+            repl=repl_func,
             string=result,
```

Comment:
This is a little eyebrow-raising for me. What's wrong with the original code?

Reply:
ahh, it's the cell-capture in a loop again. Still feels like a false-positive to me. That lambda is going nowhere outside of that line of code.

Reply:
It's a little ugly, but I'm okay to just suppress this one, since you can't accidentally break it later by assigning `value`. I'll restore it and verify that this was the error it triggered.

Reply:
Added a suppression and went with the original code.